src/status_im2/contexts/shell/animation.cljs

```diff
@@ -175,7 +175,7 @@
 
 (defn bottom-tab-on-press
   [stack-id]
-  (when-not (= stack-id @selected-stack-id)
+  (when (and @shared-values-atom (not= stack-id @selected-stack-id))
```

Comment:
could you pls elaborate on @shared-values-atom?

Reply:
As we change selected tabs with the help of animations, it is making sure we have shared values initialized before modifying them.
(Until then now, it was not required because the bottom tab functionality was kind of sandboxed, but now bottom tab can be changed for anywhere `:shell/change-tab`)